quarkchain/cluster/prom.py

```diff
@@ -86,14 +87,11 @@ def prometheus_balance(args):
             # Rpc not ready, wait and try again.
             time.sleep(3)
             continue
-        for token_name, (time_stamp, balance_dict) in total_balance.items():
-            balance_gauge.labels(
-                latest_block_height, time_stamp, "total", token_name
-            ).set(sum(balance_dict.values()))
+        for token_name, (_, balance_dict) in total_balance.items():
+            balance_gauge.labels("total", token_name).set(sum(balance_dict.values()))
+            block_height_gauge.set(latest_block_height)
```

Comment:
In this way, you cannot map a specific height to the balance it has, but assume that block may have the balance with the same time series fetched by Prometheus?

Reply:
correct. we are missing balances for certain heights. our current code doesn't promise all blocks are queried

Reply:
Yes, our work is more like "sampling" the block height and get their balance, so we are actually performing a "fake" real-time monitoring. In this way, there is no need to match them one by one